operator/duties/proposer.go

```diff
@@ -106,6 +106,15 @@ func (h *ProposerHandler) HandleDuties(ctx context.Context) {
 	}
 }
 
+func (h *ProposerHandler) HandleInitialDuties(ctx context.Context) {
+	slot := h.network.Beacon.EstimatedCurrentSlot()
+	epoch := h.network.Beacon.EstimatedEpochAtSlot(slot)
+	h.processFetching(ctx, epoch, slot)
+	h.fetchFirst = false
+	h.indicesChanged = false
+	h.logger.Info("fetching duties on init done")
```

Comment:
either add more info to this log or remove it. at least make it debug. IMO we can have one log about initial duties fetched either on `operatorNode` or on the `scheduler`